asian-pacific-journal-of-tropical-medicine.csl

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Asian Pacific Journal of Tropical Medicine</title>
+    <title-short>APJTM</title-short>
+    <id>http://www.zotero.org/styles/asian-pacific-journal-of-tropical-medicine</id>
+    <link href="http://www.zotero.org/styles/asian-pacific-journal-of-tropical-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/european-journal-of-human-genetics" rel="template"/>
+    <link href="http://www.apjtm.net/news/anthors.htm" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <category field="biology"/>
+    <issn>1995-7645</issn>
+    <eissn>2352-4146</eissn>
+    <updated>2017-03-19T18:53:39+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always" delimiter-precedes-et-al="never"/>
+      <label strip-periods="true" prefix=", "/>
```

Comment:
Don't strip the period here -- it's OK if it duplicates with the period from the suffix, since citeproc will switch it (and it's part of the abbreviation)